src/Components/Popover.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { cloneElement } from "react"
+import styled, { StyledFunction } from "styled-components"
+
+import * as fonts from "../Assets/Fonts"
+
+interface PopoverArrowPositionProps {
+  arrowOffsetTop?: string
+  arrowOffsetLeft?: string
+}
+
+interface PopoverProps
+  extends React.HTMLProps<HTMLDivElement>,
+    PopoverArrowPositionProps {
+  placement?: "top" | "right" | "bottom" | "left"
+}
+
+export class Popover extends React.Component<PopoverProps> {
+  private readonly Arrows = {
+    top: <PopoverTopArrow />,
+    right: <PopoverRightArrow />,
+    bottom: <PopoverBottomArrow />,
+    left: <PopoverLeftArrow />,
+  }
+
+  render() {
+    const {
+      placement,
+      arrowOffsetLeft,
+      arrowOffsetTop,
+      ref,
+      ...props
+    } = this.props
+
+    return (
+      <PopoverContainer {...props}>
+        {cloneElement(this.Arrows[placement], {
+          arrowOffsetLeft,
+          arrowOffsetTop,
+        })}
+
+        <PopoverInner>{this.props.children}</PopoverInner>
+      </PopoverContainer>
+    )
+  }
+}
+
+const PopoverContainer = styled.div`
+  text-align: left;
+  text-align: start;
```

Comment:
I think you meant to only have one `text-align`?

Reply:
Yeah, this looks weird. Maybe we don't need both of them so it'll just default to whatever is set to the entire `<body>`.